src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -222,6 +223,113 @@ void should_return_500_with_application_problem_response_when_assign_call_fails(
             ));
     }
 
+    @ParameterizedTest
+    @CsvSource(value = {
+        "IA, Asylum",
```

Comment:
Are we still writing test for IA?

Reply:
Not sure, since all tests have IA coverage, if we cleanup, then we do it for all

Reply:
actioned